updateChangeLog (#34852)
Azure/azure-sdk-for-java: sdk/cosmos/azure-cosmos/CHANGELOG.md

* Added configurability for minimum connection pool size for all containers through a system property - `COSMOS.MIN_CONNECTION_POOL_SIZE_PER_ENDPOINT` - See [PR 33983](https://github.com/Azure/azure-sdk-for-java/pull/33983).
* Added `CosmosContainerProactiveInitConfigBuilder:setAggressiveWarmupDuration(Duration aggressiveWarmupDuration)` public API to switch between aggressively opening connections
  in a blocking manner to defensively opening connections in a non-blocking manner after `aggressiveWarmupDuration` has elapsed - See [PR 33983](https://github.com/Azure/azure-sdk-for-java/pull/33983).

> * Added end to end timeout policy for Document/Item operations. Request will get cancelled if the request can not finish before timeout - See [PR 34554](https://github.com/Azure/azure-sdk-for-java/pull/34554).

Reviewer: ```suggestion
* Added end to end timeout policy for Item operations. Requests will be cancelled if they have not finished before the configured timeout - See [PR 34554](https://github.com/Azure/azure-sdk-for-java/pull/34554).
```

Author: updated
